data/battle_scripts_1.s

```diff
@@ -1749,6 +1749,7 @@ BattleScript_CopycatFail:
 	ppreduce
 	goto BattleScript_ButItFailed
 
+@ restoreattacker and restoretarget are used to obtain attacker and target of the called move
```

Comment:
The comment is probably redundant since the tests already will avoid a regression.